framework/include/loops/ThreadedElementLoopBase.h

```diff
@@ -260,10 +260,14 @@ ThreadedElementLoopBase<RangeType>::operator()(const RangeType & range, bool byp
           postElement(elem);
           continue;
         }
+        // To skip some look-ups, we look up the element first
+        const bool any_bdy_to_consider = _mesh.elementMayHaveASideOnABoundary(elem);
```

Comment:
this could be 4-6% less runtime based on the profile, but it's not working as expect